contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -588,7 +515,7 @@ contract StakeManager is IStakeManager {
                 validators[validatorId].activationEpoch <= currentEpoch) &&
             (validators[validatorId].deactivationEpoch == 0 ||
                 validators[validatorId].deactivationEpoch > currentEpoch) &&
-            validators[validatorId].status == Status.Active); // Todo: reduce logic
```

Comment:
Do you think we should actually reduce the logic instead of todos?

Reply:
tried to do it but no luck, depending on slashing we may be able to remove status var.